packages/grafana-ui/src/components/uPlot/Plot.tsx

```diff
@@ -27,6 +29,7 @@ type UPlotChartState = {
  * Exposes context for uPlot instance access
  */
 export class UPlotChart extends React.Component<PlotProps, UPlotChartState> {
+  private canvasPollTs: number = performance.now();
```

Comment:
do we want to use performace.now()?  for 300ms... the extra precision (and cost) does not really matter does it?